src/firebase/firebase.ts

```diff
@@ -0,0 +1,20 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore, doc, setDoc, Firestore } from "firebase/firestore"; 
+import dotenv from 'dotenv';
+dotenv.config();
+
+const firebaseConfig = {
+  apiKey: process.env.FIREBASE_API_KEY,
```

Comment:
Check in an env file or yaml file that contains all the keys you've used so far, with empty values.

That way, there is no need to browse the entire repository to determine which keys need to be provided and from where.